src/datasources/results/query-handlers/query-steps/QueryStepsDataSource.ts

```diff
@@ -83,16 +83,20 @@ export class QueryStepsDataSource extends ResultsDataSourceBase {
     returnCount = false
   ): Promise<QueryStepsResponse> {
     try {
-      const response = await this.post<QueryStepsResponse>(`${this.queryStepsUrl}`, {
-        filter,
-        orderBy,
-        descending,
-        projection,
-        take,
-        resultFilter,
-        continuationToken,
-        returnCount,
-      });
+      const response = await this.post<QueryStepsResponse>(
+        `${this.queryStepsUrl}`,
```

Comment:
```suggestion
        this.queryStepsUrl,
```
nit: Not related the current change, but the templating is not needed